src/main/resources/db/changelog/changes/v1.1.0/insert-instances-definition.xml

```diff
@@ -0,0 +1,120 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="MODFQMMGR-129" author="mweaver@ebsco.com" runOnChange="true">
+    <preConditions onFail="CONTINUE">
+      <viewExists viewName="src_inventory_instance"/>
+    </preConditions>
+    <comment>Add the Instance entity type</comment>
+
+    <!-- Insert the row if it doesn't exist, then update it separately.
+         This is purely for the sake of readability. This relies on ${tenant_id}, so it has to be done
+         in an XML changeset file (or YAML/JSON/whatever. Pretty much anything other than a .sql). By splitting it up,
+         the JSON blob can be an XML tag's body, rather than having to be nested inside a string, making it easier to
+         use syntax highlighting (JSON within XML = reasonably easy to work with. JSON within SQL with XML = not so
+         much) ¯\_(ツ)_/¯
+         This also makes it easy to keep this idempotent
+     -->
+    <sql>
```

Comment:
Technically, I could have done this with an `<insert>` block, but there's not a nice way to do the `ON CONFLICT`. Splitting it into 2 pieces like this makes it easy to keep this idempotent. The other nice alternative would be to do it all in a .sql file, since apparently string interpolation in .sql files works now